aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -48,7 +48,9 @@ module aptos_framework::genesis {
         // Initialize the aptos framework account. This is the account where system resources and modules will be
         // deployed to. This will be entirely managed by on-chain governance and no entities have the key or privileges
         // to use this account.
-        let (aptos_framework_account, framework_signer_cap) = account::create_aptos_framework_account();
+        let (aptos_framework_account, framework_signer_cap) = account::create_framework_reserved_account(@aptos_framework);
+        let (aptos_token_account, aptos_token_signer_cap) = account::create_framework_reserved_account(@0x3);
```

Comment:
I thought we wanted 0x2, why 0x3?

Reply:
token is deployed to 0x3. previous discussion was 0x2 is reserved for library in future

Reply:
Okay sounds good